src/chinks/VerticalAddition/VerticalAddition.java

```diff
@@ -0,0 +1,97 @@
+
+package chinks.VerticalAddition;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class VerticalAddition {
+   public class Node{
+        int data;
+        Node left;
+        Node right;
+        
+        public Node (int a)
+        {
+            this.left=null;
+            this.data=a;
+            this.right=null;
+        }
+    }
+     Node root=null;
+    
+     public void push(int a)
+    {
+        root=insert(root,a);
+    }
+    
+    public Node insert(Node root,int a)
+    {
+        if(root==null)
+        {
+            root= new Node(a);
+            return root;
+        }
+               
```

Comment:
remove newline

Reply:
does new line has to be removed even when a new function is declared in next line?